lnwire/onion_message.go

```diff
@@ -0,0 +1,79 @@
+package lnwire
+
+import (
+	"bytes"
+	"io"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+)
+
+// OnionMessage is a message that carries an onion-encrypted payload.
+// This is used for BOLT12 messages.
+type OnionMessage struct {
+	// BlindingPoint is the route blinding ephemeral pubkey to be used for
+	// the onion message.
+	BlindingPoint *btcec.PublicKey
```

Comment:
should we rename it to `PathKey` to be consistent with the latest version of the spec? It will also help clarity once the actual blob contains blinded path info

Reply:
perhaps we should also explain more re what it is used for. `an intermediate node expects an encrypted_recipient_data which it can decrypt into an encrypted_data_tlv using the path_key which it is handed along with the onion message.`